auto-self-nominate/README.md

```diff
@@ -0,0 +1,19 @@
+### Auto self nominate
+
+Automatically send nominate transaction.
+
+### Environment variables
+
+| Key                             | Description                                                                              | Example values                              | Default value |
+| ------------------------------- | ---------------------------------------------------------------------------------------- | ------------------------------------------- | ------------- |
+| NETWORK_ID                      | Network ID of connected CodeChain                                                        | cc, bc, tc                                  | null          |
+| RPC_URL                         | CodeChain HTTP JSON RPC URL                                                              | http://localhost:8080                       | null          |
+| NEED_NOMINATION_UNDER_TERM_LEFT | Send nomination when nomination will be ended after NEED_NOMINATION_UNDER_TERM_LEFT term | 2                                           | 2             |
```

Comment:
Send nomination if the nomination will end after the NEED_NOMINATION_UNDER_TERM_LEFT term

Reply:
Aha, "end" is a verb!